src/extension.ts

```diff
@@ -80,7 +96,7 @@ export async function activate(context: vscode.ExtensionContext) {
 		function parseAndDecorate(): MessageList {
 			let [messageList, errors] = parser.parse(editor!.document.getText())!;
 			decorator.decorate(editor!, messageList);
-			diagnostics.diagnose(editor!, messageList, errors);
+			diagnostics.diagnose(editor!, messageList, errors);quickfixes.update(messageList);
```

Comment:
nit: newline between statements

Reply:
I am used to the great autoformat support in the Dart VS Code extension ;)